src/Spd.Manager.Licence/BizProfileContract.cs

```diff
@@ -83,6 +84,7 @@ public record BizProfileUpdateRequest
     public Guid? SoleProprietorLicenceId { get; set; }
     public string? SoleProprietorSwlPhoneNumber { get; set; }
     public string? SoleProprietorSwlEmailAddress { get; set; }
+    public ContactInfo? BizManagerContactInfo { get; set; }

```

Comment:
do not add here. LoginResponse does not need BizManagerContactInfo

Reply:
I think you got confused with collapsed view. If you expand it, you will see that `BizManagerContractInfo` is defined within `BizProfileUpdateRequest` and not `BizUserLoginResponse`